x-pack/test/api_integration/deployment_agnostic/apis/observability/apm/service_maps/utils.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { APIReturnType } from '@kbn/apm-plugin/public/services/rest/create_call_apm_api';
+import { GroupResourceNodesResponse } from '@kbn/apm-plugin/common/service_map';
+import { ServiceMapSpan } from '@kbn/apm-plugin/common/service_map/types';
+
+export function getElements({ body }: { body: APIReturnType<'GET /internal/apm/service-map'> }) {
+  if ('elements' in body) {
```

Comment:
Couldn't this simply be `body.elements ?? []`? Relying on `in` will have effects like `{ body: undefined }` returning `undefined` instead of `[]` because the object *has* an `elements` property, but its value is `undefined`. Is this intended?

Reply:
This was intended to return both `elements` or `spans`, but I decided for two different functions and left this like that. Will adjust it.